src/target/arc_mem.c

```diff
@@ -55,11 +55,9 @@ static int arc_mem_read_block(struct target *target, uint32_t addr,
 	assert(!(addr & 3));
 	assert(size == 4);
 
-	/* Always call D$ flush, it will decide whether to perform actual
-	 * flush. */
-	CHECK_RETVAL(arc32_dcache_flush(target));
+	CHECK_RETVAL(arc32_cache_flush(target));
 	CHECK_RETVAL(arc_jtag_read_memory(&arc32->jtag_info, addr, count, buf,
-		    arc_mem_is_slow_memory(arc32, addr, size, count)));
+		     arc_mem_is_slow_memory(arc32, addr, size, count)));
```

Comment:
If you insist on fixing this line, you should change indent to tabs, if I understand correctly, func arguments should be indented by two tabs over the function call.